Assets/HoloToolkit/SpectatorView/Scripts/Utils/ARMarkerController.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity.SpectatorView
```

Comment:
I have to be that guy, but can we update this folder to be `Utilities`?  No reason for a folder to be abbreviated

Reply:
Also, let's align the namespaces to match the folder structure.
`HoloToolkit.Unity.SpectatorView.Utilities`